lms/static/sass/multicourse/_home.scss

```diff
@@ -12,12 +12,10 @@ $course-search-input-height: ($button-size);
 
   > header {
     @include linear-gradient($homepage__header--gradient__color--alpha, $homepage__header--gradient__color--bravo);
+    @include clearfix();
     background-size: cover;
     background-image: $homepage-bg-image;
-    border-bottom: 1px solid $border-color-3;
     box-shadow: 0 1px 0 0 $course-header-bg, inset 0 -1px 5px 0 $shadow-l1;
-    @include clearfix();
-    height: 460px;
     overflow: hidden;
     margin-top: $header_image_margin;
```

Comment:
This margin needs to be removed  - it's -69px and messes up the layout in FF (but doesn't affect Chrome).